wlauto/workloads/youtube/__init__.py

```diff
@@ -0,0 +1,90 @@
+    package = 'com.google.android.youtube'
+    version = '11.19.56'
+    activity = ''
+    description = '''
+    A workload to perform standard productivity tasks within YouTube.
+
+    The workload plays a video from the app, determined by the ``video_source`` parameter.
+    While the video is playing, a some common actions are done such as video seeking, pausing
+    playback and navigating the comments section.
+
+    Test description:
+    The ``video_source`` parameter determines where the video to be played will be found
+    in the app. Possible values are ``search``, ``home``, ``my_videos``, and ``trending``.
+    -A. search - Goes to the search view, does a search for the given term, and plays the
+        first video in the results. The parameter ``search_term`` must also be provided
+        in the agenda for this to work. This is the default mode.
+    -B. home - Scrolls down once on the app's home page to avoid ads (if present, would be
+        first video), then select and plays the video that appears at the top of the list.
+    -C. my_videos - Goes to the 'My Videos' section of the user's account page and plays a
+        video from there. The user must have at least one uploaded video for this to work.
+    -D. trending - Goes to the 'Trending Videos' section of the app, and plays the first
+        video in the trending videos list.
+
+    For the selected video source, the following test steps are performed:
+    1.  Navigating to the general app settings page to disable autoplay. This improves test
+        stability by preventing
+    2.  Jumping forward in the video by touching the video seek bar and letting the video play
+        for a few seconds.
+    3.  Expanding the info card that shows video metadata, then collapsing it again.
+    4.  Scrolling down to the end of related videos and comments under the info card, and then
+        back up to the beginning. A maximum of 5 swipe actions is performed in either direction.
+    '''
+
+    parameters = [
+        Parameter('video_source', kind=str, default='search',
+                  allowed_values=['home', 'my_videos', 'search', 'trending'],
+                  description='''
+                  Determines where to play the video from. This can either be from the
+                  YouTube home, my videos section, trending videos or found in search.
+                  '''),
+        Parameter('search_term', kind=str,
+                  default='Big Buck Bunny 60fps 4K - Official Blender Foundation Short Film',
+                  description='''
+                  The search term to use when ``video_source`` is set to ``search``. Ignored otherwise.
+                  '''),
+    ]
+
+    view = [
+        package + '/com.google.android.apps.youtube.app.WatchWhileActivity',
+        package + '/com.google.android.apps.youtube.app.honeycomb.SettingsActivity',
+    ]
+
+    def __init__(self, device, **kwargs):
+        super(Youtube, self).__init__(device, **kwargs)
```

Comment:
Are you on the latest master? 

There was a problem with earlier superclass definition where `__init__` was defined as `def __init__(self)` i.e. no `device` and `**kwargs`.

But that was fixed in the merged commit. See https://github.com/ARM-software/workload-automation/commit/530714c61cefb5323890029823731a21d7fceca3#diff-78d101373d86c8b2294558ca4ae78c67R514